mocks/mocks.py

```diff
@@ -23,6 +23,44 @@
 from mocks.utils import load_yaml_config
 
 
+def init_mocks(config):
+    """
+    Creates some initials for the code. This function will be converted to
+    the __init__ when the code uses class.
+
+    Parameters
+    ----------
+    config : dict
+        A dictionary of configuration items.
+
+    Returns
+    -------
+    config : dict
+        A dictionary of configuration items.
+    huntsman : gunagala.imager.Imager
+        Imager instance from gunagala.
+    psf_data : numpy.ndarray
+        Imager's psf data.
+    cosmo : astropy.cosmology.core.FlatLambdaCDM
+        The main cosmology that will be used by the code.
+    redshift : numpy.float
+        The redshift of the galaxy computed with respect to the distance of
+        the observer to the galaxy.
+    """
+    cosmo = create_cosmology(config)
+    redshift = compute_redshift(config['galaxy_distance'],
+                                cosmo)
+    config['pixel_scale'] = compute_pixel_scale(redshift,
+                                                cosmo,
+                                                config['sim_pc_pixel'])
+    imagers = gunagala.imager.create_imagers()
+    # In the following line the term 'canon_sbig_dark' will be changed to
+    # 'imager'. It is in progress in another PR so I prefer to wait for that.
+    huntsman = imagers['canon_sbig_dark']
+    psf_data = huntsman.psf.pixellated()
+    return config, huntsman, psf_data, cosmo, redshift
```

Comment:
You can see that `config['total_mag']` is not included here. The reason is that to compute `config['total_mag']` someone needs the data from simulation while all other stuff that is computed in this function only needs `config`.